core/src/main/resources/jenkins/monitor/EndOfLifeOperatingSystemAdminMonitor/message.properties

```diff
@@ -0,0 +1,9 @@
+End_of_Life_Heading={0} in Jenkins
+Upcoming_End_of_Life=<p class="jenkins-!-margin-bottom-0">You are running Jenkins on {0}.&nbsp;&nbsp;\
+  Jenkins will no longer support {0} after <strong>{1}</strong>.\
+  <p>Please plan your upgrade to a supported operating system <strong>before {1}</strong>.</p>\
+  <p>Refer to <a href="https://www.jenkins.io/redirect/operating-system-end-of-life" target="_blank" rel="noopener noreferrer">the documentation</a> for details.</p>
```

Comment:
should a query parameter include the OS maybe?

Reply:
Good suggestion.

I don't see any query parameter support in the redirect use on jenkins.io,. but I do see a very effective use of a fragment at the end of the URL.  When reporting a bug, we can append the Jira component number to the `report-issue` URL and it will pass that component number to the final destination

For example https://www.jenkins.io/participate/report-issue/redirect/#21710  becomes a link that will report a bug, enhancement, or security issues to the workflow-aggregator-plugin.  

I could do something similar here by adding some form of the operating system name and version as a fragment at the end of the redirect URL.  Initially it will likely be ignored, but in the future we could use the fragment for navigation within the page.

Reply:
it would allow analytics to be done if it was a query parameter 😄

Reply:
Implemented in 24d991a6e5011ecc38b81da19f8b7d4fcb6d41f7